src/components/MaterialUI/DataDisplay/TooltippedElements/TooltippedTypography.test.js

```diff
@@ -1,22 +1,34 @@
 import React from "react";
-import { shallow } from "enzyme";
+import { shallow, mount } from "enzyme";
 import TooltippedTypography from "./TooltippedTypography";
 
 describe("TooltippedTypography", () => {
-  it("Passes parameters in HOC correctly", () => {
-    const children = "text";
-    const titleValue = "tooltip title";
-    const className = "class";
+	it("should contain the forwarded ref in the child span", () => {
+		const value = "tooltip title";
 
-    const tooltippedTypography = (
-      <TooltippedTypography children={children} className={className} titleValue={titleValue} />
-    );
+		const component = mount(<TooltippedTypography value={value} />);
```

Comment:
shallow should be enough here btw.

As you maybe noticed I started use shallow as much as possible coz it executes much faster. And since we are having more than 1000 tests now in orc-shared (and probably we gonna have 5 times more to the moment when OMS product will be done) it make sense to start think about saving execution time for tests.

So, if possible to use shallow for test then better use it instead of mount.

Reply:
I tried with shallow, but test coverage is not 100% in that case, adding a simple test like I did brought it back to 100%. It does not actually hurt, still, I understand your point.